_rules/invalid-form-field-value-36b590.md

```diff
@@ -0,0 +1,227 @@
+---
+id: 36b590
+name: Error message describes invalid form field value
+rule_type: atomic
+description: |
+  This rule checks that text error messages provided when the user completes a form field with invalid values or using an invalid format identify the cause of the error or how to fix the error.
```

Comment:
They might (it's subjective, right?). But for the description, I believe it's best to keep both so that people looking specifically for one or the other will be able to find it.